config/api.yml

```diff
@@ -2464,47 +2464,55 @@
     :collection_actions:
       :get:
       - :name: read
-        :identifier: policy_profile_view
+        :identifier: miq_policy_set_view
       :post:
       - :name: query
-        :identifier: policy_profile_view
+        :identifier: miq_policy_set_view
       - :name: create
-        :identifier: profile_new
+        :identifier: miq_policy_set_new
       - :name: edit
-        :identifier: profile_edit
+        :identifier: miq_policy_set_edit
       - :name: delete
-        :identifier: profile_delete
+        :identifier: miq_policy_set_delete
     :resource_actions:
       :get:
       - :name: read
-        :identifier: policy_profile_view
+        :identifier: miq_policy_set_view
       :post:
       - :name: edit
-        :identifier: profile_edit
+        :identifier: miq_policy_set_edit
       - :name: delete
-        :identifier: profile_delete
+        :identifier: miq_policy_set_delete
       :delete:
       - :name: delete
-        :identifier: profile_delete
+        :identifier: miq_policy_set_delete
       :patch:
       - :name: edit
-        :identifier: profile_edit
+        :identifier: miq_policy_set_edit
       :put:
       - :name: edit
-        :identifier: profile_edit
+        :identifier: miq_policy_set_edit
     :policies_subcollection_actions:
       :post:
       - :name: assign
-        :identifier: policy_profile_assign
+        :identifier:
+        - miq_policy_conditions_assignment
+        - miq_policy_events_assignment
```

Comment:
This was the reason why I asked this question to @h-kataria 

https://github.com/ManageIQ/manageiq/pull/20438/files#r525530422

In my PR, I think I was a bit confused, and am not sure I did the right thing:

https://github.com/ManageIQ/manageiq-api/pull/965/files#diff-0f4eeb68b6115c7dcee6074cbf57d5e60b4f0abd7abe5b2ec5b1f8cab58d8d35R2498

So that probably should not match mine, but we probably should determine what is the correct permission identifier here.  I would assume it would be under the `miq_policy_set*` based on the other permissions, but perhaps I am missing something.  That said, the PR that changed all these in core didn't really provide much context for me to go off of.

cc @gtanzillo @h-kataria

Reply:
The `miq_policy_set_view` is definitely not enough, as you're modifying the policy profile by assigning conditions or events to it. Also these are both exposed in the UI...